terraform/03_gke_cluster.tf

```diff
@@ -146,13 +146,33 @@ resource "null_resource" "install_istio" {
   depends_on = [null_resource.set_gke_context]
 }
 
+# Get GKE cluster's default service credentials through project_number
+# Solves Issue #372. When upstream bug is resolved, remove this.
+resource "null_resource" "get_credentials" {
+  provisioner "local-exec" {
+    command = "gcloud iam service-accounts keys create key.json --iam-account $(gcloud projects list --filter='${data.google_project.project.project_id}' --format='value(PROJECT_NUMBER)')-compute@developer.gserviceaccount.com"
```

Comment:
Is there terraform provider for iam? If so, it's better to use it as opposed to a gcloud. However it can also be solved separately, after this PR is merged.